languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -35467,6 +35467,20 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_TOURS_CRITIQUES" name="Tours critiqués divers (barbarismes, impropriétés, solécismes, etc.)">
+      <rule id="DE_MANIERE_A_CE_QUE" name="de manière à ce que" type="style" tags="picky" default="temp_off">
```

Comment:
Looks good! I think the picky tag isn't needed, you can take it out when you activate the rule.